src/accounts/context/userAccount.tsx

```diff
@@ -83,32 +86,25 @@ export const UserAccountProvider: FC<Props> = React.memo(({children}) => {
 
   const handleGetAccounts = useCallback(async () => {
     try {
-      const resp = await getAccounts({})
-      if (resp.status !== 200) {
-        // set user account status to error;...TODO
-        throw new Error(resp.data.message)
+      const accounts = await getUserAccounts()
+
+      setUserAccounts(accounts)
+      const defaultAcctArray = accounts.find(line => line.isDefault)
+      if (defaultAcctArray) {
+        const defaultId = defaultAcctArray[0].id
+        setDefaultAccountId(defaultId)
       }
-      const {data} = resp
-      if (Array.isArray(data)) {
-        setUserAccounts(data)
-
-        const defaultAcctArray = data.filter(line => line.isDefault)
-        if (defaultAcctArray && defaultAcctArray.length === 1) {
-          const defaultId = defaultAcctArray[0].id
-          setDefaultAccountId(defaultId)
-        }
 
-        // isActive: true is for the currently logged in/active account
-        const activeAcctArray = data.filter(line => line.isActive)
-        if (activeAcctArray && activeAcctArray.length === 1) {
-          const activeId = activeAcctArray[0].id
-          setActiveAccountId(activeId)
-        }
+      // isActive: true is for the currently logged in/active account
+      const activeAcctArray = accounts.find(line => line.isActive)
+      if (activeAcctArray) {
+        const activeId = activeAcctArray[0].id
+        setActiveAccountId(activeId)
       }
     } catch (error) {
       event('multiAccount.retrieveAccounts.error', {error})
     }
-  }, [dispatch, defaultAccountId])
```

Comment:
I don't see `dispatch` or `defaultAccountId` in here - was this a mistake, or were these present before?

Is there anything else missing from the useEffect dependency array that would create a new warning?

Reply:
they were extraneous. my changes seem to be working fine but i will test my changes again.